subsys/nrf_rpc/include/nrf_rpc_utils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef UTILS_NRF_RPC_UTILS_H_
+#define UTILS_NRF_RPC_UTILS_H_
+
+/**
+ * @defgroup nrf_rpc_utils nRF RPC utilities.
+ * @{
+ * @brief nRF RPC utilities functions.
+ *
+ */
+
+/** @brief Get version of remote server the RPC client is connected to.
+ *
+ *  @retval version of the remote on success.
+ *  @retval NULL on failure.
+ */
+const char *nrf_rpc_get_version(void);
```

Comment:
```suggestion
const char *nrf_rpc_get_ncs_commit_sha(void);
```
as this is what it really is. The doc must be clear on which version this returns as there are multiple version strings available.